test/images/scripts/lib/atomic.setup

```diff
@@ -18,16 +18,29 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
+set -e
+
+# The docker pool should grow automatically as needed, but we grow it
+# explicitly here anyway.  This is hopefully more reliable.
+systemctl start docker
+lvresize atomicos/root        -l+50%FREE -r
+lvresize atomicos/docker-pool -l+100%FREE
+
 # Configure kubernetes. Since this writes to /etc have to do before upgrade
 /var/lib/testvm/kubernetes.setup
 
 # fully upgrade host first
 atomic host upgrade
 
-# Checkout current os branch
-# we'll use it every time we build a new tree
+# Checkout the just upgraded os branch since we'll use it every time
+# we build a new tree.
+#
+# HACK - Find a better way to compute the ref.
+# https://lists.projectatomic.io/projectatomic-archives/atomic-devel/2016-July/msg00015.html
+
+remote=$(ostree remote list)
 ref=$(python -c 'import json; print json.load(open("/usr/share/rpm-ostree/treefile.json"))["ref"]')
-ostree checkout "$ref" /var/local-tree
+ostree checkout "$remote:$ref" /var/local-tree
```

Comment:
This seems to break the OSTree test case. Or am I misunderstanding the test results.